packages/web/src/components/edit-collection/EditCollectionForm.tsx

```diff
@@ -157,6 +158,12 @@ export const EditCollectionForm = (props: EditCollectionFormProps) => {
             <VisibilityField
               entityType={isAlbum ? 'album' : 'playlist'}
               isUpload={isUpload}
+              isAllowedToPublish={
+                isAlbum ||
```

Comment:
why do we check `isAlbum` separately? shouldn't we also check that albums have more than 0 tracks?

Reply:
I remember something about removing this check, but tbh I don't remember the logic. I think it has to do with hidden/scheduled tracks being possible now?